tests/unittest/_torch/attention/test_attention_mla.py

```
@@ -685,8 +791,12 @@ def yarn_get_mscale(scale=1, mscale=1):
                     ],
                 ),
                 mapping=mapping,
-                enable_flash_mla=torch.cuda.get_device_capability() == (9, 0),
             )
+            if backend_name == "TRTLLM":
+                gen_metadata_kwargs[
+                    'enable_flash_mla'] = torch.cuda.get_device_capability(
+                    ) == (9, 0)
```

Flake8 (7.3.0)

[error] closing bracket does not match indentation of opening bracket's line

(E123)